exercises/complex-number/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct Complex {
+    // Add in the fields
+}
+
+impl Complex {
+    pub fn new(_re: f64, _im: f64) -> Complex {
+        unimplemented!()
+    }
+
+    pub fn exp(&self) -> Complex {
+        unimplemented!()
+    }
+
+    // Return the real part
```

Comment:
All the doc comments look good, but they should begin with triple forward slashes (`///`) so that `rustdoc` can use them.